troposphere/static/js/components/badges/MyBadges.react.js

```diff
@@ -45,7 +63,6 @@ define(function (require) {
       if(myBadges.length == 0){
         return (
           <div className="mine">
-            <h4 onClick={this.check}>Check badges</h4>
             You haven't earned any badges yet.
           </div>
```

Comment:
I think this text needs to be in a template string, so: 

```
   {`You haven't earned any badges yet.`}
</div>
``` 

This will make it so the unmatching `'` doesn't cause GitHub heartburn...